### PR TITLE
Add information on setting GPU compute mode to wICE guide

### PR DESCRIPTION
**Pull request by stevenvdb**
Choosing the GPU compute mode through Slurm should now work, this PR documents how this can be done.

### CODE REVIEW

**Comment by stevenvdb** [comment]
ping @moravveji @mselwa

**Review by mselwa — APPROVED** [review]
Looks good.

File: source/leuven/wice_advanced_guide.rst [hunk 1]
```diff
@@ -163,6 +163,55 @@ Also keep in mind that applying your ``~/.bashrc`` settings in your Slurm jobs
...
+----------------------------
+
+NVIDIA GPUs support multiple `compute modes
+<https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#compute-modes>`_.
+By default, the compute mode is set to `Exclusive-process` on our clusters
+(which is the best setting in the majority of cases), but you can choose
+another compute mode at job submission time. This is done by making use of a
+plugin for our Slurm job scheduler:
+
+::
```
Comment (lexming) [inline source/leuven/wice_advanced_guide.rst:178]: I suggest to replace all the basic `::` with a code block to get syntax highlighting:
```suggestion
.. code-block:: shell
```

File: source/leuven/wice_advanced_guide.rst [hunk 2]
```diff
@@ -163,6 +163,55 @@ Also keep in mind that applying your ``~/.bashrc`` settings in your Slurm jobs
...
+   ...
+   Options provided by plugins:
+   
+         --gpu_cmode=<shared|exclusive|prohibited>
+                                 Set the GPU compute mode on the allocated GPUs to
+                                 shared, exclusive or prohibited. Default is
+                                 exclusive
+
+Submitting a batch job where you want to set the compute mode of your NVIDIA
+GPU(s) to `shared` can be done with:
```
Comment (moravveji) [inline source/leuven/wice_advanced_guide.rst:190]: ... to be `shared` can be ...

File: source/leuven/wice_advanced_guide.rst [hunk 3]
```diff
@@ -163,6 +163,55 @@ Also keep in mind that applying your ``~/.bashrc`` settings in your Slurm jobs
...
+
+::
+
+   srun --ntasks-per-node=9 --nodes=1 --gpus-per-node=1 --account=<YOUR_ACCOUNT> \
+        --cluster=wice --time=01:00:00 --partition=gpu --gpu_cmode=shared \
+        --pty /bin/bash -l
+
+A few notes on this features:
+
+* To check the behaviour is as expected, execute ``nvidia-smi`` in your job
```
Comment (moravveji) [inline source/leuven/wice_advanced_guide.rst:206]: ... in your job.

**Review by moravveji — APPROVED** [review]
Looks good to me.
Two minor suggestions are given.

File: source/leuven/wice_advanced_guide.rst (hunk 1 above)
Comment (stevenvdb) [reply source/leuven/wice_advanced_guide.rst]: Done in 0ad90459ed4001b9ae4c855d7af487dac0801a99

File: source/leuven/wice_advanced_guide.rst (hunk 2 above)
Comment (stevenvdb) [reply source/leuven/wice_advanced_guide.rst]: Done in 5eced4f65aab5a9e73dbd6a715c27ff247a67a1b.

File: source/leuven/wice_advanced_guide.rst (hunk 3 above)
Comment (stevenvdb) [reply source/leuven/wice_advanced_guide.rst]: Done in 5eced4f65aab5a9e73dbd6a715c27ff247a67a1b.

**Pull request merged**